src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -124,5 +125,10 @@ Integer getInt(Map<Option, ?> options) {
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
 
-  // TODO(ajaykannan): implement "Organization" functionality when available (issue #319)
+  /**
+   * Fetches an Organization by the specified resource name.
```

Comment:
`Fetches an Organization by the specified resource name.`  => `Fetches an Organization for the specified resource name.`

Reply:
done